queries/dune/queries.ts

```diff
@@ -100,3 +107,39 @@ export const getAgentsTotalsData = async () => {
 
   return firstRow ? (firstRow as AgentsLeaderboardTotals) : null;
 };
+
+const DUNE_OMEN_TRADE_METRICS = 4576765;
+
+export const getOmenTradeMetrics = async ({
+  page = 1,
+  pageSize = 50,
+  sort_by = 'profit_loss desc',
+  filters = '',
+}) => {
+  const limit = pageSize;
+  const offset = (page - 1) * pageSize;
+  const filtersString = filters ? `&filters=${filters}` : '';
+  console.log('filtersString:', filtersString);
+  const url = `https://api.dune.com/api/v1/query/${DUNE_OMEN_TRADE_METRICS}/results?limit=${limit}&offset=${offset}&sort_by=${sort_by}${filtersString}`;
```

Comment:
Maybe we can abstract this into a utils fn, WDYT?

Reply:
I think url is okay here for now, I can put it in a variable, but since it's only used here I'd rather have it in this file.

Reply:
Yes!

I meant to create a util fn in this file to take the query ID, limit, offset, sortby and filters string. No prob!

Reply:
Yes, did some abstractions since the code was being reused.